.buildkite/docker-build-push.nix

```diff
@@ -0,0 +1,73 @@
+# This script will load nix-built docker images of cardano-rest applications
+# into the Docker daemon (must be running), and then push to the Docker Hub.
+# Credentials for the hub must already be installed with # "docker login".
+#
+# There is a little bit of bash logic to replace the default repo and
+# tag from the nix-build (../nix/docker.nix).
+#
+# 1. The repo (default "inputoutput/cardano-explorer-api") is changed to match
+#    the logged in Docker user's credentials.
+#
+# 2. The tag (default "VERSION") is changed to reflect the
+#    branch which is being built under this Buildkite pipeline.
+#
+#    - All commits are tagged with commit hash.
+#    - If the branch is master then master tag is updated
+#    - If there is a tag associated with the commit then that tag is pushed as well
+
+{ nodePackages ?  import ../. {}
```

Comment:
should be named something other than nodePackages restPackages or maybe just localPackages?